### PR TITLE
feat(klabel, kooltip): added support for max-width and tooltip-attributes [KHCP-3486]

### PR DESCRIPTION
**Pull request by LukeSwierlik**
# Summary
Task in Jira: https://konghq.atlassian.net/browse/KHCP-3486

## Vue 3

**Did you create a corresponding PR to add this change to the `beta` branch? (required)**

- [ ] **Yes**, here is a link to the PR: `[link to beta branch PR]`
- [ ] **No**, I did not create a PR for the `beta` branch and I have added the `port to beta branch` tag to this PR.
- [ ] **No**, because `[type your reasons]`

If you have questions, tag `@adamdehaven` or `@kaiarrowood`.

---

## PR Checklist

- [ ] Does not introduce dependencies
- [ ] **Functional:** all changes do not break existing APIs and if so, bump major version.
- [ ] **Tests pass:** check the output of yarn test packages/<Kongponent>
- [ ] **Naming:** the files and the method and prop variables use the same naming conventions as other Kongponents
- [ ] **Framework style:** abides by the essential rules in Vue's style guide
- [ ] **Cleanliness:** does not have formatting issues, unused code (e.g., console.logs), or leftover comments
- [ ] **Docs:** includes a technically accurate README, uses JSDOC where appropriate

### CODE REVIEW

File: packages/KoolTip/KoolTip.vue [hunk 1]
```diff
@@ -83,6 +88,7 @@ export default {
   },
   mounted () {
     this.className = this.$el && this.$el.className
+    // this.maxWidth = this.maxWidth === 'auto' ? this.maxWidth : this.maxWidth + 'px'
```
Comment (kaiarrowood) [inline packages/KoolTip/KoolTip.vue:91]: Are you planning on using this code eventually?

File: packages/KPop/KPop.vue [hunk 2]
```diff
@@ -176,6 +176,10 @@ export default {
       type: String,
       default: '200'
     },
+    maxWidth: {
```
Comment (kaiarrowood) [inline packages/KPop/KPop.vue:179]: This prop is not documented?

Comment (LukeSwierlik) [reply packages/KPop/KPop.vue:179]: Thx! Fixed

**Review by adamdehaven — CHANGES_REQUESTED** [review]

**Review by kaiarrowood — APPROVED** [review]
LGTM!

**Pull request merged**